packages/data-stores/src/site/types.ts

```diff
@@ -43,3 +43,28 @@ export interface CreateSiteParams {
 		theme?: string;
 	};
 }
+
+export interface ExistingSiteDetails {
+	ID: number;
+	name: string;
+	description: string;
+	URL: string;
+}
+
+export interface ExistingSiteError {
+	error: string;
+	message: string;
+}
+
+export type ExistingSiteResponse = ExistingSiteDetails | ExistingSiteError;
+
+export type MetadataDispatch = {
+	invalidateResolution: (
```

Comment:
It [seems](https://github.com/WordPress/gutenberg/blob/4857ad58c1241b3d63d21a6880c989b85746c3dc/packages/block-library/src/embed/settings.js#L114) like `invalidateResolution` also exists as a generic action in the `core/data` store (akin to `isResolving`), so we might not have to expose it here :thinking:

Reply:
I've updated this to use `core/data` 👍